package.json

```diff
@@ -157,16 +153,13 @@
     "merge-stream": "2.0.0",
     "mocha": "10.2.0",
     "mq-polyfill": "1.1.8",
-    "normalize.css": "8.0.1",
```

Comment:
It looks like the mention of `normalize.css` [here](https://github.com/uswds/uswds/blob/develop/packages/usa-search/src/styles/_usa-search.scss#L115) could be updated or removed. It's possible it could be updated to `"uswds-elements/lib/normalize"`, but looking at https://github.com/uswds/uswds/blob/develop/packages/uswds-elements/lib/_normalize.scss, it doesn't look like anything is actually being overridden.

Reply:
The [`appearance: none`](https://github.com/uswds/uswds/blob/4ed09988d695412b48ff1909203de56e4f3b7b09/packages/usa-search/src/styles/_usa-search.scss#L119) would still override the [`-webkit-appearance: textfield`](https://github.com/uswds/uswds/blob/4ed09988d695412b48ff1909203de56e4f3b7b09/packages/uswds-elements/lib/_normalize.scss#L293) from the `_normalize.scss` stylesheet. I think the comment itself is meant to serve as an explanation for [the disabled Stylelint rule](https://github.com/uswds/uswds/blob/4ed09988d695412b48ff1909203de56e4f3b7b09/packages/usa-search/src/styles/_usa-search.scss#L117), since Stylelint would normally expect a selector of `[type="search"]`, which would not have enough specificity to take priority over the normalized styles.

Reply:
But yes I would agree with your point that the comment could be updated to reference USWDS's internal normalization stylesheet rather than `normalize.css`, which is not used directly.